services/agents/registry.go

```diff
@@ -1103,6 +1105,36 @@ func (r *Registry) StartPTSummaryAction(ctx context.Context, id, pmmAgentID stri
 	return nil
 }
 
+// StartPTPgSummaryAction starts pt-pg-summary action on the pmm-agent.
+// The pt-pg-summary may require some of the following params: host, port, username, password.
```

Comment:
unnecessary comment and also wrong, there is no host in params

Reply:
> My comment was not addressed nor discussed.
> 
> I don't see point why in comment describe everything again what is already in function signature.
> Code should be self-documented, and don't comment what is already clear. Also these types of comments are typically unmaintained after while.

The comment was updated per your request. This comment serves as the function description. And in Go they sometime sound stupid because there is a certain rule for the beginning.
This comment says what params the called program may require. The functions can return nil in case of error, in this case it returns nil when ok. Nevertheless it depends on how the function is designed. I don't have to look into the function to know that so the comment saves my time. I don't see any problem of this comment.
Some people don't write comments, some people don't write diacritics and some people don't write at all they just emoji.